code/modules/clothing/masks/gasmask.dm

```diff
@@ -143,6 +200,7 @@
 	resistance_flags = FLAMMABLE
 	actions_types = list(/datum/action/item_action/adjust)
 	species_exception = list(/datum/species/golem)
+	tint = 0
```

Comment:
you forgot to set the tint to 0 for sexyclown masks, sexymime masks, monkey masks, etc.

Reply:
Going to remove the tint changes from this pr